index.php

```diff
@@ -56,3 +56,21 @@
 
 $mediaWiki = new MediaWiki();
 $mediaWiki->run();
+
+// SEO-586 Log what pages link to index.php URLs on 5% of page views
+if ( rand( 0, 99 ) < 5 ) {
+	$scriptUrl = $_SERVER[ 'SCRIPT_URL' ] ?? 'empty';
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "empty" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)